publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -416,15 +427,17 @@ private boolean hasExchange() throws RemRemPublishException {
         try {
             channel = connection.createChannel();
         } catch (final IOException e) {
-            log.info("Exchange " + exchangeName + " does not Exist");
+            log.info("Exchange " + exchangeName + " does not exist");
             throw new RemRemPublishException("Exception occurred while creating Channel with Rabbitmq connection ::" + factory.getHost() + factory.getPort() + e.getMessage());
```

Comment:
The original exception should be passed to RemRemPublishException constructor. Not only here, but anywhere the exception is created.